spec/support/fixtures/test/main.swift

```diff
@@ -11,13 +11,13 @@ class SomeClass: RedundantProtocol {
     private var assignOnlyProperty = 0
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/b9634e3f794e9d32353be741484f2602179a3ff4/spec/support/fixtures/test/main.swift#L11"></span>Property 'assignOnlyProperty' is assigned, but never used</td>
    </tr>
  </tbody>
</table>